integration-tests/mongodb-panache/src/test/java/io/quarkus/it/mongodb/panache/resolvers/BaseMongodbPanacheResolversTest.java

```diff
@@ -0,0 +1,92 @@
+package io.quarkus.it.mongodb.panache.resolvers;
```

Comment:
It doesn't look like there's a test case where no `Resolver` has been defined.  Since we're injecting that mock, it's always going to find a `Resolver`, yes?

Reply:
Yes, that`s it. This no `Resolver` case is already covered by the other tests since we`re not implementing it in the other test classes, but if needed I can create a test class only to cover this scenario/

Reply:
Duh.  You're right.  Nevermind me.  :)